docs/development/remote.md

```diff
@@ -181,13 +181,27 @@ you prefer for development in general.
 
 If you use [TextMate](https://macromates.com), Atom, VS Code, or a
 similar GUI editor, tools like
+[Remote - SSH](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-ssh) and
```

Comment:
This should probably be more obviously scoped to be a Visual Studio Code only thing.

E.g.